sdk/trustedsigning/arm-trustedsigning/package.json

```diff
@@ -57,46 +63,47 @@
     ]
   },
   "dependencies": {
-    "@azure-rest/core-client": "^2.1.0",
+    "@azure/core-util": "^1.12.0",
+    "@azure-rest/core-client": "^2.3.1",
     "@azure/abort-controller": "^2.1.2",
     "@azure/core-auth": "^1.9.0",
     "@azure/core-lro": "^3.1.0",
-    "@azure/core-rest-pipeline": "^1.19.0",
-    "@azure/core-util": "^1.11.0",
-    "@azure/logger": "^1.1.4",
+    "@azure/core-rest-pipeline": "^1.20.0",
+    "@azure/logger": "^1.2.0",
     "tslib": "^2.8.1"
   },
   "devDependencies": {
-    "@azure-tools/test-credential": "workspace:^",
-    "@azure-tools/test-recorder": "workspace:^",
-    "@azure-tools/test-utils-vitest": "workspace:^",
-    "@azure/dev-tool": "workspace:^",
-    "@azure/eslint-plugin-azure-sdk": "workspace:^",
-    "@azure/identity": "catalog:internal",
-    "@types/node": "catalog:",
-    "@vitest/browser": "catalog:testing",
-    "@vitest/coverage-istanbul": "catalog:testing",
-    "dotenv": "catalog:testing",
-    "eslint": "catalog:",
-    "playwright": "catalog:testing",
-    "typescript": "catalog:",
-    "vitest": "catalog:testing"
+    "@azure-tools/test-credential": "^2.0.0",
```

Comment:
This should probably be reverted back to workspace specifiers (and I wonder if we need to update the codegen as well? cc @jeremymeng )